tests/lsdb_macauff/import_pipeline/create_catalogs.py

```diff
@@ -0,0 +1,40 @@
+"""Script to convert generated test catalogs into hipscatted catalogs.
+
+See also lsdb_mc_data.py, for generating some data using macauff utils."""
+
+import hipscat_import.pipeline as runner
+from hipscat_import.catalog.arguments import ImportArguments
+
+
+def create_catalogs():
```

Comment:
Did you mean to leave the `input_file_list` and `output_path` pointing to your local directories?